packages/plugins/whisper-geth/src/whisperGethClient.js

```diff
@@ -157,6 +156,7 @@ class WhisperGethClient {
     let rpcApi = this.config.rpcApi;
     let wsApi = this.config.wsApi;
     let args = ['--ipcdisable']; // Add --ipcdisable as ipc is not needed for Whisper and it conflicts on Windows with the blockchain node
+    args.push(`--port="${this.communicationConfig || 30304}"`);
```

Comment:
I think I messed up when making a suggestion. It should be this instead:
```suggestion
    args.push(`--port="${this.communicationConfig.connection.port || 30304}"`);
```